Package.swift

```diff
@@ -14,7 +14,7 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/tuist/ProjectAutomation", .exact("3.2.0")),
-        .package(url: "https://github.com/realm/SwiftLint", .exact("0.46.5")), // it is a core dependency of the plugin, the version should be under control and locked
+        .package(url: "https://github.com/realm/SwiftLint", revision: "e497f1f"), // v0.47.1, commit SHA due to its use of unsafe flags
```

Comment:
Can you check if it's still a problem in `0.48.0`? Have you tried opening an issue/discussion on the SwiftLint repo about this?

Reply:
This will still happen with 0.48.0. This exists in other SPM packages that need to link statically against SwiftSyntax such as Sourcery as well. Currently, this involves unsafe build flags in SPM and SPM only allows you to link by commit/branch when there are unsafe build flags.

If you wish to keep the version based match here, it will remain locked at 0.46.5, which feels like a problem if the plugin isn't using latest SwiftLint version.

Reply:
GTM

Reply:
still, let's bump to the latest version :D